lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -1,5 +1,6 @@
 import time
 import datetime
+from http import HTTPStatus
 from pbench.server.database.models.users import User
```

Comment:
We should probably have blank lines around the import that you added, and the bare `import`'s should be reordered.